website/react-magma-docs/src/components/MainNav/index.js

```diff
@@ -29,20 +29,19 @@ const activeStyleInverse = {
 const Heading2 = styled.h2`
   align-items: center;
   display: flex;
-  font-size: ${magma.typeScale.size05.fontSize};
-  font-weight: 600;
-  line-height: ${magma.typeScale.size05.lineHeight};
+  font-size: ${magma.typeScale.size04.fontSize};
+  font-weight: 500;
+  line-height: ${magma.typeScale.size06.lineHeight};
   justify-content: space-between;
   margin: 0;
-  padding: ${magma.spaceScale.spacing03} ${magma.spaceScale.spacing06}
-    ${magma.spaceScale.spacing04};
+  padding: ${magma.spaceScale.spacing03} ${magma.spaceScale.spacing06};
 `;
 
 const Heading3 = styled.h3`
   color: ${props =>
-    props.isInverse ? magma.colors.neutral07 : magma.colors.neutral};
-  font-size: ${magma.typeScale.size04.fontSize};
-  font-weight: 700;
+    props.isInverse ? magma.colors.neutral06 : magma.colors.neutral700};
```

Comment:
Can we replace `neutral06` with one of the new colors here?

Reply:
Yeah, you bet.  Sorry about the package-lock.  I didn't have that problem in the past, but I'll make sure I don't include it from now on.